src/api.js

```diff
@@ -51,6 +51,18 @@ export const removeVotes = (review_id) => {
     });
 };
 
+export const getCategories = () => {
+  return gamesReviewsApi.get("/categories").then((res) => {
+    return res.data.categories
+  })
+}
+
+export const getReviewsByCategory = (category) => {
+  return gamesReviewsApi.get(`/reviews?category=${category}`).then((res) => {
```

Comment:
we can make this more DRY by altering your get request for all reviews to take a params object with the selected category value (google axios params get request for more info) - this would mean you won't need to do an if condition in the reviews component